cms/templates/base.html

```diff
@@ -118,5 +118,6 @@
     </script>
     <%include file="widgets/segment-io-footer.html" />
     <div class="modal-cover"></div>
+    <%static:optional_include_mako file="zendesk-widget.html" is_theming_enabled="True" />
```

Comment:
If we have to fork/extend the platform template, we shouldn't do so w/ a special case here.
Instead, we should add some form of a broad include that we can further extend later, as needed.

We could call it `body_post.html` or some such.
Regardless, we should PR this upstream.

Reply:
I want to avoid needed to add _another_ include here later, should we decide to add more to the page.